packages/core/src/constants.ts

```diff
@@ -1,5 +1,27 @@
 import { GriffelStylesUnsupportedCSSProperties, LookupItem, SequenceHash } from './types';
 
+// ----
+
+// Heads up!
+// These constants are global and will be shared between Griffel instances
+// Any change in them should happen only in a MAJOR version
+
+const __GLOBAL__: any = typeof window === 'undefined' ? global : window;
+const __NAMESPACE_PREFIX__ = '@griffel/';
+
+function getGlobalVar<T>(name: string, defaultValue: T): T {
+  return (__GLOBAL__[Symbol.for(__NAMESPACE_PREFIX__ + name)] =
```

Comment:
nit: returning assignments is a complete mind**** for devs that don't come from js background, I'd rather avoid that